charts/fluent-operator/values.yaml

```diff
@@ -268,6 +268,7 @@ fluentd:
   crdsEnable: true
   enable: false
   name: fluentd
+  mode: "collector"
```

Comment:
```suggestion
  # valid modes include "collector" and "agent"
  # The "collector" mode will deploy Fluentd as a StatefulSet as before
  # The new "agent" mode will deploy Fluentd as a DaemonSet
  mode: "collector"
```

Reply:
@karan56625 we'd better add this comments to values as well :)

Reply:
done